jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskprocesscontext/TaskProcessContextPresenter.java

```diff
@@ -33,16 +34,20 @@
 import org.jbpm.workbench.ht.service.TaskService;
 import org.jbpm.workbench.pr.events.ProcessInstancesWithDetailsRequestEvent;
 import org.jbpm.workbench.pr.service.ProcessRuntimeDataService;
+import org.kie.workbench.common.workbench.client.PerspectiveIds;
 import org.uberfire.client.mvp.Activity;
 import org.uberfire.client.mvp.ActivityManager;
 import org.uberfire.client.mvp.PlaceManager;
 import org.uberfire.client.mvp.UberView;
 import org.uberfire.mvp.impl.DefaultPlaceRequest;
+import org.uberfire.security.ResourceRef;
+import org.uberfire.security.authz.AuthorizationManager;
+import org.uberfire.workbench.model.ActivityResourceType;
 
 @Dependent
 public class TaskProcessContextPresenter {
 
-    public static final String PROCESS_INSTANCE_DETAILS = "DataSet Process Instances With Variables";
+    public static final String PROCESS_INSTANCE_DETAILS = "DataSetProcessInstancesWithVariables";
```

Comment:
@nmirasch should this string match what is defined in the perspective? https://github.com/droolsjbpm/jbpm-wb/blob/master/jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/perspectives/DataSetProcessInstancesWithVariablesPerspective.java#L39

Also, I can see some other places where this is also defined using the original value, see:
https://github.com/droolsjbpm/kie-wb-distributions/blob/master/kie-wb/kie-wb-webapp/src/main/resources/security-module-user.properties#L20
https://github.com/droolsjbpm/kie-wb-distributions/blob/master/kie-wb/kie-wb-webapp/src/test/java/org/kie/workbench/security/WorkbenchACLTest.java#L104